dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi12/fat/tests/AroundConstructTestBase.java

```diff
@@ -105,9 +107,11 @@ public static void setUp() throws Exception {
                         .add(new FileAsset(new File("test-applications/postConstructErrorMessageApp.war/resources/WEB-INF/beans.xml")), "/WEB-INF/beans.xml")
                         .addAsLibrary(utilLib);
 
+        server.setMarkToEndOfLog(server.getDefaultLogFile());
         ShrinkHelper.exportDropinAppToServer(server, aroundConstructApp);
         ShrinkHelper.exportDropinAppToServer(server, postConstructErrorMessageApp);
-        server.waitForStringInLogUsingMark("CWWKZ0001I.*Application aroundConstructApp started");
+        assertNotNull("aroundConstructApp started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*aroundConstructApp"));
+        assertNotNull("postConstructErrorMessageApp started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*postConstructErrorMessageApp"));
```

Comment:
see... you've checked for both here!

Reply:
I was more on the ball when I changed this file :wink: